python/pypto/ir/compile.py

```
@@ -20,6 +22,38 @@
+    if not resolved_bin:
+        raise FileNotFoundError(
+            "ptoas binary not found in PATH. Please install the ptoas package: pip install ptoas"
+        )
+
+    cmd = [resolved_bin, pto_path, "-o", output_path]
+    if ptoas_flags:
+        cmd.extend(ptoas_flags)
+
+    result = subprocess.run(cmd, capture_output=True, text=True, check=False)
+    if result.returncode != 0:
+        raise RuntimeError(f"ptoas compilation failed: {result.stderr.strip()}")
```

Ruff (0.15.2)

[warning] Avoid specifying long messages outside the exception class

(TRY003)